doc_source/create-kubeconfig.md

```diff
@@ -32,6 +34,9 @@ users:
         - "<cluster-name>"
         # - "-r"
         # - "<role-arn>"
+      # env:
+        # - name: AWS_PROFILE
+        # - value: "<awsprofile>"
```

Comment:
the dash in front of value is incorrect, it should be
env:
 - name: AWS_PROFILE
    value: "<awsprofile>"